### DISCUSSION TITLE
Issue With Vercel Image Optimisations Using Signed Image URLs (Supabase)

### DISCUSSION THREAD

**Discussion by jackeverydayzero** [vercel/community#2395] [Help] [answered]
### Summary

Hey All!
Been using Vercel and Supabase to deploy an application. Love the duo, but i've recently hit the image optimisations limit (weird because we're only developing).
Found that this has happened because when I request an image (from the Supabase CDN), it is signed and has a token attached to the URL.
Is it possible to get Vercel to only optimise images without certain query string params? I can't remove them in the loader because the loader runs before the image is requested.

Advice would be appreciated. We can't upgrade to Pro with this issue because we would really quickly exceed 5000 optimised images.

Example here where an image is requested 4 times. It is the same image, but it just has a different token:
```
https://bfzvqpxfaoqteoamifvy.supabase.co/storage/v1/object/sign/stands/stand-1/main_contaminants/spago-bowl.png?token=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1cmwiOiJzdGFuZHMvc3RhbmQtMS9tYWluX2NvbnRhbWluYW50cy9zcGFnby1ib3dsLnBuZyIsImlhdCI6MTY4MzY5NDIyMSwiZXhwIjoxNjg0Mjk5MDIxfQ.84al3akxnXO5tkOs9YG_9kB2it9YJW5vM_fNCRThaVY

https://bfzvqpxfaoqteoamifvy.supabase.co/storage/v1/object/sign/stands/stand-1/main_contaminants/spago-bowl.png?token=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1cmwiOiJzdGFuZHMvc3RhbmQtMS9tYWluX2NvbnRhbWluYW50cy9zcGFnby1ib3dsLnBuZyIsImlhdCI6MTY4MzY5NDM4OCwiZXhwIjoxNjg0Mjk5MTg4fQ.BogcDAAyb9xx5BGnbVBp8iH8E5QzMAHod8BAwDG5x54

https://bfzvqpxfaoqteoamifvy.supabase.co/storage/v1/object/sign/stands/stand-1/main_contaminants/spago-bowl.png?token=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1cmwiOiJzdGFuZHMvc3RhbmQtMS9tYWluX2NvbnRhbWluYW50cy9zcGFnby1ib3dsLnBuZyIsImlhdCI6MTY4MzcwNTA1MSwiZXhwIjoxNjg0MzA5ODUxfQ.TShNIq8ojJ2t9gO99798j_A_qXBuIcqT38racfDJi0E

https://bfzvqpxfaoqteoamifvy.supabase.co/storage/v1/object/sign/stands/stand-1/main_contaminants/spago-bowl.png?token=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1cmwiOiJzdGFuZHMvc3RhbmQtMS9tYWluX2NvbnRhbWluYW50cy9zcGFnby1ib3dsLnBuZyIsImlhdCI6MTY4MzcwNTA3NiwiZXhwIjoxNjg0MzA5ODc2fQ.78AnVuzbTC-dcg7pZInNrlg31rylA3IFoGkfKZiyC7g

```

Jack.

### Additional information

It's a similar issue to this -> https://github.com/vercel/next.js/discussions/31601

**Comment by amyegan** [comment, marked as answer]
Hi @jackeverydayzero! If you can't get an image URL without a unique token, you can use the `unoptimized` prop on the Next.js `<Image/>` component or in the `next.config.js` file to [disable image optimization](https://nextjs.org/docs/app/api-reference/components/image#unoptimized) on some or all of your images.

More options are described here: [How to minimize Image Optimization costs](https://vercel.com/docs/concepts/image-optimization/managing-image-optimization-costs#how-to-minimize-image-optimization-costs)

**Reply by jackeverydayzero** [reply]
No worries, thanks Amy.
